hack/istio/install-istio-via-sail.sh

```diff
@@ -196,3 +198,8 @@ spec:
       - name: otel-tracing
     randomSamplingPercentage: 100
 EOF
+
+# deploy istio-ingressgateway 
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+echo $SCRIPT_DIR
+kubectl apply -n istio-system -f "${SCRIPT_DIR}"/istio-gateway.yaml
```

Comment:
This should move to `install-multi-primary.sh` script after we install sail. Not all callers of this script want to install this gateway but the multi-cluster install does. When you do move it you'll need to call it twice: once for each cluster and use `ISTIO_NAMSPACE` instead of the hard coded `istio-system`.

Reply:
moved into `install-multi-primary.sh`